cypress/e2e/functional/document_tests/exemplar_test_spec.js

```diff
@@ -0,0 +1,109 @@
+import SortedWork from "../../../support/elements/common/SortedWork";
+import ClueCanvas from '../../../support/elements/common/cCanvas';
+import DrawToolTile from '../../../support/elements/tile/DrawToolTile';
+import TextToolTile from "../../../support/elements/tile/TextToolTile";
+
+let sortWork = new SortedWork,
+  clueCanvas = new ClueCanvas,
+  drawToolTile = new DrawToolTile,
+  textToolTile = new TextToolTile;
+
+// This unit has `initiallyHideExemplars` set, and an exemplar defined in curriculum
+const queryParams1 = `${Cypress.config("qaConfigSubtabsUnitStudent5")}`;
+const exemplarName = "Ivan Idea: First Exemplar";
+
+function beforeTest(params) {
+  cy.clearQAData('all');
+  cy.visit(params);
+  cy.waitForLoad();
+}
+
+function drawSmallRectangle(x, y) {
+  drawToolTile.getDrawToolRectangle().last().click();
+  drawToolTile.getDrawTile().last()
+  .trigger("mousedown", x, y)
+  .trigger("mousemove", x+25, y+25)
+  .trigger("mouseup", x+25, y+25);
+}
+
+function addText(x, y, text) {
+  drawToolTile.getDrawToolText().last().click();
+  drawToolTile.getDrawTile().last()
+    .trigger("mousedown", x, y)
+    .trigger("mouseup", x, y);
+  drawToolTile.getDrawTile().last()
+    .trigger("mousedown", x, y)
+    .trigger("mouseup", x, y);
+  drawToolTile.getTextDrawing().get('textarea').type(text + "{enter}");
+}
```

Comment:
The above two functions (drawSmallRectangle & addText) are pretty generic: perhaps they should be moved to cypress/support/elements/tile/DrawToolTile.js?